params.go

```diff
@@ -0,0 +1,33 @@
+package wotsp
+
+import (
+	"crypto"
+)
+
+// Opts groups the parameters required for WOTSP operations.
+//
+// Opts implements crypto.SignerOpts and crypto.DecrypterOpts.
+type Opts struct {
+	Mode    Mode
+	PubSeed []byte
+	Address Address
+}
+
+// mode should implement crypto.SignerOpts
+var _ crypto.SignerOpts = Opts{}
```

Comment:
Maybe it's worth making it an /internal/wotsp/ part of a post-quantum crypto library, as opposed to a separate library. Otherwise one could argue that even if used as a one-time only Signer, it is still a Signer, and it could use the interface implementation.

That said, it seems the /x/crypto/ packages don't tend to implement the standard library interfaces either.